reaktive/src/jvmJsCommonMain/kotlin/com/badoo/reaktive/utils/serializer/SerializerImpl.kt

```diff
@@ -9,6 +9,7 @@ internal abstract class SerializerImpl<in T>(queue: Queue<T>) : Serializer<T> {
     private val monitor = Any()
     private var isDraining = false
 
+    @Suppress("ReturnCount")
```

Comment:
What about this one? Do we really want it?

Reply:
`return X` count in one method.